api/router/info.go

```diff
@@ -0,0 +1,32 @@
+package router
+
+import (
+	"net/http"
+)
+
+type InfoHTTPHandler interface {
+	GetInfo(http.ResponseWriter, *http.Request)
+}
+
+func (r Router) setupInfoRoutes(infoServer InfoHTTPHandler) {
+	// swagger:route GET /api/resource resource getResourceRequest
```

Comment:
Would it be alright if we could remove this? This is for `info`. Not `resource`

Reply:
removed.

I think `info` was something from when I was setting up the API originally.  It just returns a helloWorld.  So, I will remove it.